patches/minecraft/net/minecraft/block/StemBlock.java.patch

```diff
@@ -30,7 +30,7 @@
           }
  
        }
-@@ -101,4 +103,10 @@
+@@ -98,4 +100,10 @@
```

Comment:
I'd just discard these changes since you didn't touch anything here.

Reply:
This has been done.